test/e2e/cluster_extension_install_test.go

```diff
@@ -759,16 +759,6 @@ func TestClusterExtensionRecoversFromInitialInstallFailedWhenFailureFixed(t *tes
 		if assert.NotNil(ct, cond) {
 			assert.Equal(ct, metav1.ConditionTrue, cond.Status)
 			assert.Equal(ct, ocv1alpha1.ReasonRetrying, cond.Reason)
-		}
-	}, pollDuration, pollInterval)
-
-	t.Log("By eventually failing to install the package successfully due to insufficient ServiceAccount permissions")
-	require.EventuallyWithT(t, func(ct *assert.CollectT) {
-		assert.NoError(ct, c.Get(context.Background(), types.NamespacedName{Name: clusterExtension.Name}, clusterExtension))
-		cond := apimeta.FindStatusCondition(clusterExtension.Status.Conditions, ocv1alpha1.TypeInstalled)
-		if assert.NotNil(ct, cond) {
-			assert.Equal(ct, metav1.ConditionFalse, cond.Status)
-			assert.Equal(ct, ocv1alpha1.ReasonFailed, cond.Reason)
 			assert.Contains(ct, cond.Message, "forbidden")
```

Comment:
Seems odd that this line was kept? It was part of the `EventuallyWithT()` that was deleted; not sure it should be added the the previous one?

Reply:
I think this is correct here. The logic behind moving it to the previous one is that we no longer set the `Installed` condition failure here, but rather reflect the failure in the `Progressing` status condition (which is the condition we are checking immediately before this. In this case we expect the `Progressing` condition to eventually be `True`, with a reason `Retrying`, and a message that signals that there were insufficient permissions attached to the `ServiceAccount` (asserted by looking for the "forbidden" string in the message). 

I could be wrong here, but I believe we want to ensure we aren't progressing further in the test until we've identified that the failure is related to permissions and this should do that.